.github/workflows/continous-delivery.yml

```
@@ -71,6 +96,20 @@ jobs:
       environment: prod
       tag: "v${{ github.run_number }}"
       region: "osc-secnum-fr1"
-    secrets:
-      SCALINGO_API_TOKEN: ${{ secrets.SCALINGO_API_TOKEN }}
-      DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+      version-new-features: ${{fromJson(needs.deploy-staging.outputs.release-notes).body}}
+      secret-scalingo-api-token: ${{ secrets.SCALINGO_API_TOKEN }}
+      secret-discord-webhook-url: ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+  create-release-notes:
+    name: Create release notes
+    runs-on: ubuntu-latest
+    needs: [ deploy-staging, deploy-prod ]
```

Juste la prod ca doit suffire 
```suggestion
    needs: [ deploy-prod ]
```

---

@JeromeBu on a besoin de ce needs pour accéder aux variables créées sur `deploy-staging`